src/sum.py

```diff
@@ -1,2 +1,11 @@
-def sum(a: int, b: int) -> int:
-    return a + b
+def sum() :
```

Comment:
questa funzione non ritorna niente invece ci si aspetta faccia la somma tra due valori in input imho

definisci almeno il valore di ritorno nella funzione (ovvero None, quindi `-> None:`)